java/jooby/pom.xml

```diff
@@ -15,7 +15,7 @@
     <!-- Startup class -->
     <application.class>benchmark.App</application.class>
 
-    <jooby.version>[3.11,3.12)</jooby.version>
+    <jooby.version>4.0.3</jooby.version>
```

Comment:
Is it safe to keep range @jknack ?

I mean specifying a range could be good (to decrease maintainance), but maybe we good use a non-production ready version (like a rc, or milestone)